tests/plugins/dut_monitor/dut_monitor.py

```diff
@@ -0,0 +1,89 @@
+import argparse
+import time
+import os
+
+from  datetime import datetime
+
+
+FETCH_CPU_CMD = "ps -eo pcpu,args | sort -k 1 -r | tail -n +2"
+FETCH_HDD_CMD = "df -hm /"
+MEASURE_DELAY = 2
+
+
+def process_cpu(log_file):
+    """
+    @summary: Fetch CPU utilization. Write to the file total and top 10 process CPU utilization.
+    @param log_file: Opened file object to store fetched CPU utilization.
+    """
+    buff = []
+    general_template = """
+"{timestamp}":
+    total: {total}
+    top_consumer:
+"""
+    per_process_template = "        {cpu_utilization}: \"{process}\""
+    top_consumer_list = []
+    total = 0
+    top_consumer_counter = 10
+
+    for line in os.popen(FETCH_CPU_CMD).readlines():
+        process_consumed = float(line.split()[0])
+        total += process_consumed
+        if top_consumer_counter:
+            top_consumer_list.append(per_process_template.format(cpu_utilization=process_consumed, process=line.split()[1]))
```

Comment:
An example line of the output of FETCH_CPU_CMD:
```
 0.0 /bin/bash -c rm -f /var/run/rsyslogd.pid && /usr/sbin/rsyslogd -n
```
Parsing `process` from such line using `process=line.split()[1]` would only get `/bin/bash`. Maybe it is better to record more detailed process information using code like:
```
process=" ".join(line.split()[1:])
```

Reply:
Some times we can have very long process name, and it will be not to readable.
But in some cases it can help to debug. Ok will add to store full string.

Reply:
Fixed.